app/assets/javascripts/admin/datastore_health_check.js

```diff
@@ -26,14 +26,51 @@ export const pingDataStoreIfAppropriate = memoizedThrottle(async (requestedUrl:
   if (usedDataStore != null) {
     const { url } = usedDataStore;
     const healthEndpoint = `${url}/data/health`;
-    Request.triggerRequest(healthEndpoint, { doNotCatch: true, mode: "cors", timeout: 5000 }).catch(
-      () => {
-        Toast.warning(messages["datastore.health"]({ url }));
-      },
+    Request.triggerRequest(healthEndpoint, {
+      doNotInvestigate: true,
+      mode: "cors",
+      timeout: 5000,
+    }).then(
+      () => checkVersionMismatch(url),
+      () => Toast.warning(messages["datastore.health"]({ url })),
     );
   }
 }, 5000);
 
+async function checkVersionMismatch(url: string) {
+  const [buildinfoWebknossos, buildinfoDatastore] = await Promise.all([
+    RestAPI.getBuildInfo(),
+    RestAPI.getDataStoreBuildInfo(url),
+  ]);
+
+  const expectedDatastoreApiVersion = buildinfoWebknossos.webknossos.datastoreApiVersion;
+
+  const buildInfoWebknossosDatastore = buildinfoDatastore.webknossosDatastore
+    ? buildinfoDatastore.webknossosDatastore
+    : buildinfoDatastore.webknossos;
+  const suppliedDatastoreApiVersion = buildInfoWebknossosDatastore.datastoreApiVersion;
+
+  if (expectedDatastoreApiVersion.split(".")[0] < suppliedDatastoreApiVersion.split(".")[0]) {
```

Comment:
@rschwanhold I think this still does string comparison, where "12">"5" is false

Reply:
Ok, good point. I will edit that after lunch